ztp/siteconfig-generator/siteConfig/siteConfig.go

```diff
@@ -228,6 +230,43 @@ type TangConfig struct {
 	Thumbprint string `yaml:"thumbprint" json:"thp"`
 }
 
+type DiskPartition struct {
+	Device     string       `yaml:"device"`
+	Partitions []Partitions `yaml:"partitions"`
+}
+
+type Partitions struct {
+	Label         string `yaml:"label"`
```

Comment:
I was thinking about the user input here... Is it better to have them specify the label, and then we guess what the intended mount-point is from that, or would it be rather to have them specify the mount-point and we construct the proper label?

Reply:
I think mount-point would be better instead of Label as the user input. This mount-point path is also the common info between PV and MachineConfig.

Reply:
Changed the input to `mount_point` and generating the `label` instead